crates/miden-lib/src/account/auth/mod.rs

```diff
@@ -58,24 +58,27 @@ impl From<RpoFalcon512> for AccountComponent {
 ///
 /// The storage layout is:
 /// - Slot 0(value): Public key (same as RpoFalcon512)
-/// - Slot 1(value): Number of trigger procedures
+/// - Slot 1(value): [0, 0, allow_unauthorized_output_notes, num_tracked_procs]
 /// - Slot 2(map): A map with trigger procedure roots
 ///
 /// This component supports all account types.
 pub struct RpoFalcon512ProcedureAcl {
     public_key: PublicKey,
     auth_trigger_procedures: Vec<Word>,
+    allow_unauthorized_output_notes: bool,
```

Comment:
We should explain what is meant by `allow_unauthorized_output_notes` (ideally in the documentation of this struct).
In practice this means we disallow note anonymization / mixing, but as it's quite a "heavy" word, so I'd avoid referring to mixing if possible.